docs/usage.adoc

```diff
@@ -46,6 +46,19 @@ If you only want to use smart testing on `surefire` or `failsafe` plugin definit
 
 To do it you just need to set `smart.testing.apply.to` with either `surefire` or `failsafe` to just enable in one of them.
 
+=== Skipping Tests
+
+To skip running the tests for a particular project, Smart Testing extension provides support with 
+http://maven.apache.org/surefire/maven-failsafe-plugin/examples/skipping-tests.html[Maven Surefire/Failsafe 
+plugins' skip test functionality] offered by system properties `skipTests`, `skipITs` and `maven.test.skip`.
+
+Setting properties `skipTests` or `maven.test.skip` disables Smart Testing extension and exhibits standard 
+Surefire/Failsafe plugin behavior.
+
+`skipITs` when used with Smart Testing extension in default mode with both Surefire and Failsafe plugins, 
+skips only the integration tests being run by the Failsafe plugin and continues with unit test(s) execution by 
+Surefire plugin configured with Smart Testing configuration.
```

Comment:
> Surefire plugin configured with Smart Testing configuration

this seems to be quite confusing for me. IMHO, there should be mentioned that (on the contrary to `skipTests` or `maven.test.skip` cases) in this case the extension is activated.